pkg/cli/admin/release/mirror.go

```diff
@@ -213,6 +370,22 @@ func (o *MirrorOptions) Run() error {
 		return fmt.Errorf("--skip-release-image and --to-release-image may not both be specified")
 	}
 
+	if o.ApplyReleaseImageSignature {
+		if len(o.ReleaseImageSignatureToDir) != 0 {
+			return fmt.Errorf("--release-image-signature-to-dir and --apply-release-image-signature may not both be specified")
```

Comment:
`may not` is not great way to communicate that the flags are not compatible with each other.  we should use `must  not` or some explicit way to communicate.

Reply:
I 100% agree but followed the established pattern in the code to be consistent. Not only in mirror but throughout oc it uses "may not" in it's error output.